### ISSUE TITLE
adapt-contrib-narrative: accessibility issue

### ISSUE THREAD

**Issue by canstudios-nicolaw** [adaptlearning/adapt-contrib-narrative#235]
There are some accessibility issues when using this component in mobile view.

### Your environment
* 0.7.0 AT/ 4.0.1 FW
* NVDA screen reader with Chrome/Firefox/IE11

### Steps to reproduce
The strapline is not read out by screen readers on mobile. The screen reader reads out items in the following order (which seems strange):
- Item title
- Alt text for the image
- Item body

### Expected behaviour
The strapline should be read out by screen readers, the items should be read out in a logical order.

I realise that this could be a low priority issue due to it being a mobile-only accessibility issue.

**Comment by joe-allen-89** [comment]
When in mobile view the screen reader reads out strapline as a button followed by the graphic alt text, back button and finally the next button before navigating you to the next component. Focus is shifted to the content when content button is selected and popup appears. This appears to have been fixed by https://github.com/adaptlearning/adapt-contrib-core/pull/90
